app/views/officer/landing_applications/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<main class="govuk-main-wrapper">
+  <section class="container">
+    <h1>Landing applications</h1>
+
+    <%= govuk_table do |table|
+      table.with_caption(size: 'm', text: 'List of applications')
```

Comment:
Probably we only need one heading and can drop the `caption`?

Reply:
My understanding is that the caption is important for screen readers (https://www.w3.org/WAI/tutorials/tables/caption-summary/). I agree that having both feels superfluous, maybe I have it wrong...

Reply:
ah, interesting! Would be OK to either: 

- drop the H1 and style the table caption as a heading with `govuk-table__caption--l`. 
- make the table caption hidden but still available to screen readers

Reply:
But: if the table is the only thing on the page then perhaps the H1 heading is sufficient? If there were 2 tables on the page then I would understand the need for 2 captions...

Reply:
oh wait that's what they are already 😆

Reply:
Maybe we can differentiate a little more helpfully between the page heading and the purpose of the table e.g.

- page heading: "Landing application"
- table caption: "Applications to assess"

Reply:
ok, agreed 👍